test/puppetca_puppet_cert/fixtures/ca/inventory.txt

```diff
@@ -0,0 +1,4 @@
+0x0001 2018-06-09T09:30:19UTC 2023-06-09T09:30:19UTC /CN=Puppet CA: tfmdemo.beispiel.xyz
```

Comment:
Do you rely on these names or can we replace the domain with example.com?

Reply:
I believe the CRL uses `beispiel.xyz`. I happen to own this domain, so I think we're safe to use it.